diff --git a/src/onigposix.h b/src/onigposix.h
index 2af3717..2d0e40e 100644
--- a/src/onigposix.h
+++ b/src/onigposix.h
@@ -147,7 +147,7 @@ ONIG_EXTERN int  onig_set_default_syntax P_((OnigSyntaxType* syntax));
 ONIG_EXTERN void onig_copy_syntax P_((OnigSyntaxType* to, OnigSyntaxType* from));
 ONIG_EXTERN const char* onig_version P_((void));
 ONIG_EXTERN const char* onig_copyright P_((void));
-
+ONIG_EXTERN int onig_end P_((void));
 #endif /* ONIGURUMA_H */
 
 
